primitives/container-chain-genesis-data/src/lib.rs

```diff
@@ -54,8 +56,19 @@ pub mod json;
 // in polkadot.js, because `Vec<u8>` is serialized as `[12, 51, 124]` instead of hex.
 // That's why we use `serde(with = "sp_core::bytes")` everywhere, to convert it to hex.
 #[cfg_attr(feature = "std", derive(serde::Deserialize, serde::Serialize))]
-#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, scale_info::TypeInfo)]
-pub struct ContainerChainGenesisData {
+#[cfg_attr(feature = "std", serde(bound = ""))]
+#[derive(
+    DebugNoBound,
+    CloneNoBound,
+    EqNoBound,
+    DefaultNoBound,
+    PartialEqNoBound,
+    Encode,
+    Decode,
+    scale_info::TypeInfo,
+)]
+#[scale_info(skip_type_params(MaxLengthTokenSymbol))]
+pub struct ContainerChainGenesisData<MaxLengthTokenSymbol: Get<u32>> {
     pub storage: Vec<ContainerChainGenesisDataItem>,
     // TODO: make all these Vec<u8> bounded
```

Comment:
I want to add the other bounds in this PR as well, but I'm not sure how to do that. One option is to simply add them as additional generic params, but I don't like that because that will introduce a lot of noise every time this type is used:

```
ContainerChainGenesisData<MaxLengthTokenSymbol: Get<u32>, MaxLengthForkId: Get<u32>, ...>
```

The other option is to add one type parameter that can be used to get all the different bounds:

```
ContainerChainGenesisData<T: Config>
```

But I can't use the Config trait there, so I would probably need a new trait?